source/install.md

```diff
@@ -0,0 +1,245 @@
+
+Distributions of Conda/Mamba come with a **base environment**, which includes the installation of the package manager itself. We highly recommend leaving this environment alone! If you make a mistake and install software from a conflicting channel, or say "install anyway" one too many times, or sneeze while staring at your shadow at noon on midsummer's eve during an update, you could break your entire Conda/Mamba install. All of us at OpenFF have done this at least once! It's not the end of the world, since hopefully that Conda/Mamba install was [local to your user] and you can just delete it and reinstall, but it is frustrating.
+
+Instead of installing software into your base environment, we recommend creating new environments to install software into as you go. Some people prefer to have one big environment they install everything into, while others prefer a few smaller ones that each have the software needed for a particular task. Some of us even spin up a new environment every time they start a PR!
+
+A new environment can be created from a list of packages:
+
+```shell-session
+$ mamba create -n <environment name> -c conda-forge <package(s)>
+```
+
+Or from an [environment file], which can specify both channels and dependencies:
+
+```shell-session
+$ mamba env create -n <name> -f <path to .yaml file>
+```
+
+Any time you want to use software installed in an environment you have created, you must activate it. An environment will stay active until you close the terminal window or shell session:
+
+```shell-session
+$ conda activate <name>
+$ jupyter notebook .
+```
+
+[virtual environments]: conda:user-guide/concepts/environments
+[local to your user]: install_mamba
+
+### Advanced Environment Management
+
+If you prefer, you can specify the path to a virtual environment's root directory as a prefix instead of giving an environment's name:
+
+```shell-session
+$ mamba create -c conda-forge -p <prefix> <package(s)>
+$ conda activate <prefix>
+```
+
+If you specify neither the name `-n` nor the prefix `-p`, the current active environment will be used.
+
+New software can be installed in an environment by listing packages:
+
+```shell-session
+$ mamba install -n <name> -c conda-forge <package(s)>
+```
+
+Or by adding the dependencies from an environment file:
+
+```shell-session
+$ mamba env update -n <name> -f <path to .yaml file>
+```
+
+Mamba/Conda cache the packages you install, so if you've already downloaded a particular version, you can install it in as many environments as you like without having to re-download it. In fact, Mamba/Conda try their hardest to unpack each version once, and then hardlink to it from each environment to keep the storage size of environments down. Be aware of this if you like to tinker with the code of your dependencies - your changes may affect other environments!
```

Comment:
> tinker with ...

If this implies "don't muck around in `/site-packages/`" we should have stronger language against doing this. If you're referring to editable installs while switching branches between projects ... ugh, that's annoying and I keep doing it.

Reply:
Yeah referring to the former, I'll strengthen and clarify.

Reply:
Done in 31bd01f